main/modes/mode_slide_whistle.c

```diff
@@ -1700,7 +1764,7 @@ noteFrequency_t  arpModify(noteFrequency_t note, int8_t arpInterval)
 }
 
 /**
- * @brief Translate a slideWhistleal note to a color
+ * @brief Translate a musical note to a color
```

Comment:
lol

Reply:
Pay attention to your find-replaces, friends!